packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -128,6 +137,17 @@ export default {
         closeButtonCopy: {
             type: String,
             default: 'Close modal'
+        },
+
+        title: {
```

Comment:
Can you make sure you add these new props to the component `README` so that they're documented.